README.md

```diff
@@ -39,7 +39,7 @@ A sample query : `/api/v1/search/bing?query=fossasia&format=xml&num=10`
 ## Dependencies
 
 * [MongoDB](https://www.mongodb.com)
-* [Python 2.7 and 3.4+](https://python.org)
```

Comment:
You can add 3.6 here as build passes for python 3.6  https://travis-ci.org/fossasia/query-server/jobs/294388161

Reply:
It does not.  None of the scrapers load in Python 3.